packages/apps/frontera/src/routes/organizations/src/components/FinderTable/FinderTable.tsx

```diff
@@ -245,6 +245,44 @@ export const FinderTable = observer(({ isSidePanelOpen }: FinderTableProps) => {
     }
   };
 
+  const selectedIds = Object.keys(selection);
```

Comment:
please encapsulate this logic in use effect

Reply:
don't need any useEffect it's working fine

Reply:
it should be encapsulated for following reasons:
-In React, side effects should be handled in useEffect hooks.
-The code depends on several variables (selectedIds, tableType, isCommandMenuPrompted) that can change over time. We want to react to these changes and update the UI accordingly and at the same time we do not want to react and recompute on other changes
- readability and maintnance:  this is already complex component and dumping not encapsulated logic makes it harder to read and maintain (maybe this logic could be even exported to hook)

Reply:
i will encapsulate, but there is no point in that useEffect, because the finderTable component is already re-rendered due to all logic in the table component and due to the javascript all the code will be re computed and the condition with !isComandMenuPrompted will not pass and the js will move further.

Reply:
it has a point, re-renders related to sorting and filtering will not trigger this part of code so we are saving some unnecessary computations